packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -27,6 +27,18 @@ class CodeSizeScreen extends Screen {
 
   static const id = 'codeSize';
 
+  static const snapshotTabKey = Key('Snapshot Tab');
+  static const diffTabKey = Key('Diff Tab');
+
+  @visibleForTesting
```

Comment:
I think we can remove the visibleForTesting annotation since the keys are now used outside of this class, so they'd need to be public anyway

Reply:
I think for now the only keys that we need outside are snapshotTabKey and diffTabKey. Should I still remove the annotation for the rest of the keys?

Reply:
Fine to leave as is